bin/main.rb

```diff
@@ -1,2 +1,30 @@
-#!/usr/bin/env ruby
-puts 'Hello world!'
+require_relative '../lib/Player'
+require_relative '../lib/Game'
+
+puts 'Please enter player1 name: '
+player_1_name = gets.chomp
+player1 = Player.new(player_1_name, 'x')
+
+puts 'Please enter player2 name: '
+player_2_name = gets.chomp
+player2 = Player.new(player_2_name, 'o')
```

Comment:
- [ ] **OPTIONAL** Your game allows 2 players with the same name, which can be confusing, you can add a validation to check that this doesn't happen
![image](https://user-images.githubusercontent.com/53256139/91372227-b4c54200-e7d8-11ea-926a-18dd0e06d679.png)

Reply:
Thank you for offering this, we have changed it in Milestone 2 and merged it to ```game_logic``` branch.